docs/tuning_fledge.rst

```diff
@@ -762,6 +767,113 @@ When collection is enabled the following counters will be collected for the stor
       - The size of the JSON payload in the delete calls to the storage layer for the given table.
       - There is little an end user can influence regarding the payload size, however it gives an indication of bandwidth usage for the storage API.
 
+Purge
+=====
+
+The purpose of the purge processes within Fledge is to control the usage of the storage system. Fledge has two different purge process that run, each of these purges a different aspect of the storage within the system;
+
+  - **System Purge** - The system purge process is responsible for purging the logs held internally within the Fledge storage system. There are three types of log information held in the storage system; statistics, the audit trail and task execution history.
```

Comment:
- Colon instead of semi-colon after "in the storage system"
- Do you want to add the Oxford comma at the end? _the audit trail, and task execution history._